### DISCUSSION TITLE
inserting length with "values" field for continuous datatype returns missing value_ids error, worked in v2, fails in v3

### DISCUSSION THREAD

**Discussion by alihammadraza** [etsy/open-api#658] [Q&A] [answered]
While trying to update length property id 47626759838 for a listing with taxonomy id 929 I get error message saying "Missing input parameter: [value_ids]". This is a continuous property as in the length can be anything in inches and therefore should not require value_ids. 

When we GET properties for a similar listing filled using web UI, the length property is filled and the value_ids comes back with a huge number in billions. I presume etsy maintains a data set of all continuous values versus generated value_ids. However, I do not have access to any existing value_ids when I use the api to PUT length property in inches for a new listing. The fields I am using are as follows:
(
    [scale_id] => 5
    [values] => 42
    [scale_name] => Inches
)

I have tried random large numbers as value_ids which it seems does get stored somewhere in etsy since a GET request for properties against that listing comes back with the previously inserted values including both value and value_ids. However, the web ui doesn't show the length despite api results.

Simply put: value_ids should not be necessary for insertion in a listing like handmade rugs which doesn't come in specific sizes. This is a bug in v3. The length attribute (with same id) in api v2 worked just fine for the exact taxonomy without requiring value_ids.

**Comment by etsyachristensen** [comment]
@alihammadraza Looking at the seller taxonomy id 929, it does have a length property but the id for that is `18107239445`. We don't know what listing id you're attempting to update, so we can't look up the properties for the listing to verify, but the guess is that perhaps there's a custom property added called "Length" but it's not set with an open scale. How are you getting the id of `47626759838` for length? Is this coming from a `getListingProperties` call? If so, which shop and listing id?

**Comment by alihammadraza** [comment]
I used the getPropertiesByTaxonomyId by making a GET call @ https://openapi.etsy.com/v3/application/seller-taxonomy/nodes/929/properties

The following is the outcome:
<pre>
{
  "count": 28,
  "results": [
    {
      "property_id": 148789511893,
      "name": "Material multi",
      "display_name": "Material",
      "scales": [],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": false,
      "is_multivalued": true,
      "max_values_allowed": null,
      "possible_values": [
        {
          "value_id": 53,
          "name": "Acrylic",
          "scale_id": null,
          "equal_to": [
            206
          ]
        },
        {
          "value_id": 61,
          "name": "Bamboo",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 87,
          "name": "Chenille",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 102,
          "name": "Cotton",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2400,
          "name": "Cowhide",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 120,
          "name": "Faux fur",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 121,
          "name": "Faux leather",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 144,
          "name": "Hemp",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 153,
          "name": "Jute",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 161,
          "name": "Leather",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 187,
          "name": "Nylon",
          "scale_id": null,
          "equal_to": [
            206
          ]
        },
        {
          "value_id": 196,
          "name": "Paper",
          "scale_id": null,
          "equal_to": [
            3452
          ]
        },
        {
          "value_id": 206,
          "name": "Plastic",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 210,
          "name": "Polyester",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2402,
          "name": "Seagrass",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2403,
          "name": "Sheepskin",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 245,
          "name": "Silk",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 249,
          "name": "Sisal",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 288,
          "name": "Wool",
          "scale_id": null,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 200,
      "name": "Primary color",
      "display_name": "Primary color",
      "scales": [],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": 5,
      "possible_values": [
        {
          "value_id": 1213,
          "name": "Beige",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1,
          "name": "Black",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2,
          "name": "Blue",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1216,
          "name": "Bronze",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 3,
          "name": "Brown",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1219,
          "name": "Clear",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1218,
          "name": "Copper",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1214,
          "name": "Gold",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 5,
          "name": "Gray",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 4,
          "name": "Green",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 6,
          "name": "Orange",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 7,
          "name": "Pink",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 8,
          "name": "Purple",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1220,
          "name": "Rainbow",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 9,
          "name": "Red",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1217,
          "name": "Rose gold",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1215,
          "name": "Silver",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 10,
          "name": "White",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 11,
          "name": "Yellow",
          "scale_id": null,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 52047899002,
      "name": "Secondary color",
      "display_name": "Secondary color",
      "scales": [],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": 5,
      "possible_values": [
        {
          "value_id": 1213,
          "name": "Beige",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1,
          "name": "Black",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2,
          "name": "Blue",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1216,
          "name": "Bronze",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 3,
          "name": "Brown",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1219,
          "name": "Clear",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1218,
          "name": "Copper",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1214,
          "name": "Gold",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 5,
          "name": "Gray",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 4,
          "name": "Green",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 6,
          "name": "Orange",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 7,
          "name": "Pink",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 8,
          "name": "Purple",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1220,
          "name": "Rainbow",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 9,
          "name": "Red",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1217,
          "name": "Rose gold",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 1215,
          "name": "Silver",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 10,
          "name": "White",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 11,
          "name": "Yellow",
          "scale_id": null,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 47626759898,
      "name": "Width",
      "display_name": "Width",
      "scales": [
        {
          "scale_id": 7,
          "display_name": "Centimeters",
          "description": ""
        },
        {
          "scale_id": 11,
          "display_name": "Feet",
          "description": ""
        },
        {
          "scale_id": 5,
          "display_name": "Inches",
          "description": ""
        },
        {
          "scale_id": 10,
          "display_name": "Meters",
          "description": ""
        },
        {
          "scale_id": 4,
          "display_name": "Millimeters",
          "description": ""
        },
        {
          "scale_id": 9,
          "display_name": "Yards",
          "description": ""
        }
      ],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": 5,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 47626759838,
      "name": "Length",
      "display_name": "Length",
      "scales": [
        {
          "scale_id": 7,
          "display_name": "Centimeters",
          "description": ""
        },
        {
          "scale_id": 11,
          "display_name": "Feet",
          "description": ""
        },
        {
          "scale_id": 5,
          "display_name": "Inches",
          "description": ""
        },
        {
          "scale_id": 10,
          "display_name": "Meters",
          "description": ""
        },
        {
          "scale_id": 4,
          "display_name": "Millimeters",
          "description": ""
        },
        {
          "scale_id": 9,
          "display_name": "Yards",
          "description": ""
        }
      ],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": 5,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 47626759828,
      "name": "Diameter",
      "display_name": "Diameter",
      "scales": [
        {
          "scale_id": 7,
          "display_name": "Centimeters",
          "description": ""
        },
        {
          "scale_id": 11,
          "display_name": "Feet",
          "description": ""
        },
        {
          "scale_id": 5,
          "display_name": "Inches",
          "description": ""
        },
        {
          "scale_id": 10,
          "display_name": "Meters",
          "description": ""
        },
        {
          "scale_id": 4,
          "display_name": "Millimeters",
          "description": ""
        },
        {
          "scale_id": 9,
          "display_name": "Yards",
          "description": ""
        }
      ],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": 5,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 570246213604,
      "name": "Sustainability",
      "display_name": "Sustainability",
      "scales": [],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": false,
      "is_multivalued": true,
      "max_values_allowed": 3,
      "possible_values": [
        {
          "value_id": 144,
          "name": "Hemp",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 162,
          "name": "Linen",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 5009,
          "name": "Organic cotton",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 5018,
          "name": "Recycled polyester",
          "scale_id": null,
          "equal_to": [
            5017
          ]
        },
        {
          "value_id": 5005,
          "name": "Upcycled",
          "scale_id": null,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 99837394318,
      "name": "Pattern",
      "display_name": "Pattern",
      "scales": [],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": false,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [
        {
          "value_id": 395,
          "name": "Abstract",
          "scale_id": null,
          "equal_to": [
            2817
          ]
        },
        {
          "value_id": 396,
          "name": "Animal print",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2333,
          "name": "Bordered",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 410,
          "name": "Camouflage",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 411,
          "name": "Check",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 424,
          "name": "Floral",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 426,
          "name": "Geometric",
          "scale_id": null,
          "equal_to": [
            2817
          ]
        },
        {
          "value_id": 435,
          "name": "Ikat",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2335,
          "name": "Moroccan",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2339,
          "name": "Ombré",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2336,
          "name": "Oriental",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 445,
          "name": "Paisley",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2332,
          "name": "Patchwork",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2334,
          "name": "Persian",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 451,
          "name": "Plants & trees",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 452,
          "name": "Polka dot",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 460,
          "name": "Solid",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2338,
          "name": "Southwestern",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 465,
          "name": "Striped",
          "scale_id": null,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 47626759726,
      "name": "Shape",
      "display_name": "Shape",
      "scales": [],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": false,
      "is_multivalued": false,
      "max_values_allowed": 5,
      "possible_values": [
        {
          "value_id": 326,
          "name": "Circle",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 351,
          "name": "Octagon",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 353,
          "name": "Oval",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 361,
          "name": "Rectangle",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2345,
          "name": "Semicircle",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 371,
          "name": "Square",
          "scale_id": null,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 148789511877,
      "name": "Rug type",
      "display_name": "Type",
      "scales": [],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": false,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [
        {
          "value_id": 2374,
          "name": "Accent",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2373,
          "name": "Area",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 4181,
          "name": "Bath mat",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2376,
          "name": "Door mat",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2375,
          "name": "Runner",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2377,
          "name": "Rug pad",
          "scale_id": null,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 145330288652,
      "name": "Home style",
      "display_name": "Home style",
      "scales": [],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": false,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [
        {
          "value_id": 2384,
          "name": "Bohemian & eclectic",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2385,
          "name": "Coastal & tropical",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2387,
          "name": "Contemporary",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2388,
          "name": "Country & farmhouse",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2390,
          "name": "Industrial & utility",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2395,
          "name": "Rustic & primitive",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2396,
          "name": "Scandinavian",
          "scale_id": null,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 46803063641,
      "name": "Occasion",
      "display_name": "Occasion",
      "scales": [],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": false,
      "is_multivalued": false,
      "max_values_allowed": 5,
      "possible_values": [
        {
          "value_id": 2773,
          "name": "1st birthday",
          "scale_id": null,
          "equal_to": [
            19
          ]
        },
        {
          "value_id": 12,
          "name": "Anniversary",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 13,
          "name": "Baby shower",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 14,
          "name": "Bachelor party",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 15,
          "name": "Bachelorette party",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 16,
          "name": "Back to school",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 17,
          "name": "Baptism",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 18,
          "name": "Bar & Bat Mitzvah",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 19,
          "name": "Birthday",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 20,
          "name": "Bridal shower",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 21,
          "name": "Confirmation",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 26,
          "name": "Divorce & breakup",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 22,
          "name": "Engagement",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 23,
          "name": "First Communion",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 24,
          "name": "Graduation",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 25,
          "name": "Grief & mourning",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 27,
          "name": "Housewarming",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2774,
          "name": "LGBTQ pride",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 50,
          "name": "Moving",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 28,
          "name": "Pet loss",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 29,
          "name": "Prom",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 30,
          "name": "Quinceañera & Sweet 16",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 31,
          "name": "Retirement",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 32,
          "name": "Wedding",
          "scale_id": null,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 46803063659,
      "name": "Holiday",
      "display_name": "Holiday",
      "scales": [],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": false,
      "is_multivalued": false,
      "max_values_allowed": 5,
      "possible_values": [
        {
          "value_id": 35,
          "name": "Christmas",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 36,
          "name": "Cinco de Mayo",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 4562,
          "name": "Diwali",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 37,
          "name": "Easter",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 4564,
          "name": "Eid",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 38,
          "name": "Father's Day",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 39,
          "name": "Halloween",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 40,
          "name": "Hanukkah",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 4563,
          "name": "Holi",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 41,
          "name": "Independence Day",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 42,
          "name": "Kwanzaa",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 34,
          "name": "Lunar New Year",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 43,
          "name": "Mother's Day",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 44,
          "name": "New Year's",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 47,
          "name": "Passover",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 45,
          "name": "St Patrick's Day",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 46,
          "name": "Thanksgiving",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 48,
          "name": "Valentine's Day",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 49,
          "name": "Veterans Day",
          "scale_id": null,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 145330288592,
      "name": "Room",
      "display_name": "Room",
      "scales": [],
      "is_required": false,
      "supports_attributes": true,
      "supports_variations": false,
      "is_multivalued": true,
      "max_values_allowed": 5,
      "possible_values": [
        {
          "value_id": 2356,
          "name": "Bathroom",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2354,
          "name": "Bedroom",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 3946,
          "name": "Dorm",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2353,
          "name": "Entryway",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2357,
          "name": "Kids",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2350,
          "name": "Kitchen & dining",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2359,
          "name": "Laundry",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2351,
          "name": "Living room",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2358,
          "name": "Nursery",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2352,
          "name": "Office",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 2355,
          "name": "Patio & outdoor",
          "scale_id": null,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 501,
      "name": "Dimensions",
      "display_name": "Dimensions",
      "scales": [
        {
          "scale_id": 344,
          "display_name": "Inches",
          "description": "Inches"
        },
        {
          "scale_id": 345,
          "display_name": "Centimeters",
          "description": "Centimeters"
        },
        {
          "scale_id": 346,
          "display_name": "Other",
          "description": "Other"
        }
      ],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 502,
      "name": "Fabric",
      "display_name": "Fabric",
      "scales": [],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 500,
      "name": "Finish",
      "display_name": "Finish",
      "scales": [],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 503,
      "name": "Flavor",
      "display_name": "Flavor",
      "scales": [],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 505,
      "name": "Height",
      "display_name": "Height",
      "scales": [
        {
          "scale_id": 347,
          "display_name": "Inches",
          "description": "Inches"
        },
        {
          "scale_id": 348,
          "display_name": "Centimeters",
          "description": "Centimeters"
        },
        {
          "scale_id": 349,
          "display_name": "Other",
          "description": "Other"
        }
      ],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 507,
      "name": "Material",
      "display_name": "Material",
      "scales": [],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 508,
      "name": "Pattern",
      "display_name": "Pattern",
      "scales": [],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 509,
      "name": "Scent",
      "display_name": "Scent",
      "scales": [],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 100,
      "name": "TeeShirtSize",
      "display_name": "Size",
      "scales": [
        {
          "scale_id": 301,
          "display_name": "Alpha",
          "description": "Alpha"
        },
        {
          "scale_id": 327,
          "display_name": "Inches",
          "description": "Inches"
        },
        {
          "scale_id": 328,
          "display_name": "Centimeters",
          "description": "Centimeters"
        },
        {
          "scale_id": 335,
          "display_name": "Fluid Ounces",
          "description": "Fluid Ounces"
        },
        {
          "scale_id": 336,
          "display_name": "Millilitres",
          "description": "Millilitres"
        },
        {
          "scale_id": 337,
          "display_name": "Litres",
          "description": "Litres"
        },
        {
          "scale_id": 329,
          "display_name": "Other",
          "description": "Other"
        }
      ],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [
        {
          "value_id": 1166388104,
          "name": "XXS",
          "scale_id": 301,
          "equal_to": []
        },
        {
          "value_id": 1166388106,
          "name": "XS",
          "scale_id": 301,
          "equal_to": []
        },
        {
          "value_id": 1166388112,
          "name": "S",
          "scale_id": 301,
          "equal_to": []
        },
        {
          "value_id": 1166388116,
          "name": "M",
          "scale_id": 301,
          "equal_to": []
        },
        {
          "value_id": 1166388122,
          "name": "L",
          "scale_id": 301,
          "equal_to": []
        },
        {
          "value_id": 1166388126,
          "name": "XL",
          "scale_id": 301,
          "equal_to": []
        },
        {
          "value_id": 1166388128,
          "name": "XXL",
          "scale_id": 301,
          "equal_to": []
        },
        {
          "value_id": 1502041605,
          "name": "3XL",
          "scale_id": 301,
          "equal_to": []
        },
        {
          "value_id": 1502041607,
          "name": "4XL",
          "scale_id": 301,
          "equal_to": []
        },
        {
          "value_id": 1502041613,
          "name": "5XL",
          "scale_id": 301,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 510,
      "name": "Style",
      "display_name": "Style",
      "scales": [],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 511,
      "name": "Weight",
      "display_name": "Weight",
      "scales": [
        {
          "scale_id": 332,
          "display_name": "Pounds",
          "description": "Pounds"
        },
        {
          "scale_id": 331,
          "display_name": "Ounces",
          "description": "Ounces"
        },
        {
          "scale_id": 333,
          "display_name": "Grams",
          "description": "Grams"
        },
        {
          "scale_id": 334,
          "display_name": "Kilograms",
          "description": "Kilograms"
        }
      ],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 515,
      "name": "Device",
      "display_name": "Device",
      "scales": [],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [
        {
          "value_id": 5668983248,
          "name": "iPad",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 5668983250,
          "name": "iPad Mini",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 5668983258,
          "name": "iPhone 5",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 5745757057,
          "name": "iPhone 4/4S",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 5668983226,
          "name": "Kindle Fire HD",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 5668983232,
          "name": "Kindle Fire HD 8.9\"",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 5668983234,
          "name": "Kindle Paperwhite",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 5668983240,
          "name": "Samsung Galaxy S4",
          "scale_id": null,
          "equal_to": []
        },
        {
          "value_id": 5668983238,
          "name": "Samsung Galaxy S3",
          "scale_id": null,
          "equal_to": []
        }
      ],
      "selected_values": []
    },
    {
      "property_id": 513,
      "name": "Custom1",
      "display_name": "Custom Property",
      "scales": [],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [],
      "selected_values": []
    },
    {
      "property_id": 514,
      "name": "Custom2",
      "display_name": "Custom Property",
      "scales": [],
      "is_required": false,
      "supports_attributes": false,
      "supports_variations": true,
      "is_multivalued": false,
      "max_values_allowed": null,
      "possible_values": [],
      "selected_values": []
    }
  ]
}
</pre>

As you can see the Length property id returned using v3 is still 47626759838 which was the same attribute id for the old v2 taxonomy as well. In v2 the same attribute id worked flawlessly without requiring any value_ids. Could it be that you guys have created new taxonomy fields for 929 but the v3 is still erroneously returning v2 taxonomy fields? The listing in question is https://www.etsy.com/listing/1331585724/

**Reply by etsyachristensen** [reply]
Apologies! I was looking at the wrong database. You are correct. That is the right property id number. We will discuss with our Taxonomy team to find out what might be causing this issue.

**Comment by etsyachristensen** [comment, marked as answer]
@alihammadraza I just did some testing on this. If you don't include the value_ids parameter of course it throws and error. What I did find was that if you pass in the value_ids parameter but leave it blank, the system will fill in the value_id when it's added. Here's the cURL.
```
curl --location --request PUT 'https://openapi.etsy.com/v3/application/shops/12345678/listings/987654321/properties/47626759838' \
--header 'Content-Type: application/x-www-form-urlencoded' \
--header 'x-api-key: xxxxxx' \
--header 'Authorization: Bearer yyyyyy' \
--data-urlencode 'value_ids=' \
--data-urlencode 'values=58.3'
```

This returned:
```
{
    "property_id": 18107239445,
    "property_name": "Length",
    "scale_id": null,
    "scale_name": null,
    "value_ids": [
        18116626787
    ],
    "values": [
        "58.3"
    ]
}
```

**Reply by alihammadraza** [reply]
**Yes this fixes the issue, thanks.** 

I previously tried using null and 0 as value_ids but didn't try an empty string. I have also found that the same methodology ie providing an empty string works for values as well when e.g. system throws an error of missing values in case of discrete field like Rooms where only value_ids should be required.
